articles/quickstart/spa/_includes/_install_auth0js.md

```diff
@@ -1,7 +1,7 @@
 ## Install auth0.js
 
-Integrating Auth0 in your application requires the **auth0.js** library. Install it using npm or yarn.
-
+You need the auth0.js library to integrate Auth0 into your application.
+Install auth0.js.
```

Comment:
Shouldn't this `Install auth0.js.` be more like `To install auth0.js run the following script:`